app/scripts/modules/core/src/entityTag/alerts/EntityName.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import { IEntityTags } from 'core/domain/IEntityTags';
+import { NgReact } from 'core/reactShims';
+
+export interface IProps {
+  tag: IEntityTags;
+}
+
+/** Renders an entity name and it's account and region */
+export class EntityName extends React.Component<IProps, void> {
+  public render() {
+    const entityRef = this.props.tag.entityRef;
+    const { AccountTag } = NgReact;
+    if (!entityRef.account && !entityRef.region) {
+      return null;
+    }
+
+    return (
+      <div className="entityname">
+        <span className="entityref"><strong>{entityRef.entityId}</strong></span>
+        <span className="account-region">
+          {' ( '}
```

Comment:
It's a bit of both.  This way the spaces don't collapse during compile and intellij also doesn't whine.  I tried a few different styles and this was the least offensive to me, but open to alternatives